internal/runners/clean/run_win.go

```diff
@@ -129,3 +132,44 @@ func removePaths(logFile string, paths ...string) error {
 
 	return nil
 }
+
+func checkAdmin() error {
+	doAdminCheck := true
+	installedAsAdmin, err := getAdminInstall()
+	if err != nil {
+		doAdminCheck = false
+		multilog.Error("Could not check if initial installation was run as admin, error: %v", err)
+	}
+
+	isAdmin, err := osutils.IsAdmin()
+	if err != nil {
+		doAdminCheck = false
+		multilog.Error("Could not check if current user is an administrator, error: %v", err)
+	}
+
+	if doAdminCheck && installedAsAdmin && !isAdmin {
+		return locale.NewInputError("err_uninstall_privlege_mismatch")
+	}
+
+	return nil
+}
+
+func getAdminInstall() (bool, error) {
```

Comment:
I think we should centralize the behavior around storing and retrieving this state in the same package. Maybe `installation`?

Reply:
That's a really good idea!